server/src/main/java/org/elasticsearch/ingest/IngestService.java

```diff
@@ -113,8 +113,8 @@ public class IngestService implements ClusterStateApplier, ReportingService<Inge
     // are loaded, so in the cluster state we just save the pipeline config and here we keep the actual pipelines around.
     private volatile Map<String, PipelineHolder> pipelines = Map.of();
     private final ThreadPool threadPool;
-    private final IngestMetric totalMetrics = new IngestMetric();
-    private final List<Consumer<ClusterState>> ingestClusterStateListeners = new CopyOnWriteArrayList<>();
+    private final IngestMetric totalMetrics;
+    private final List<Consumer<ClusterState>> ingestClusterStateListeners;
```

Comment:
Looking at the constructors, I don't think we need this change since they're never anything other than what these previously were?